stdlib/experimental/mqtt/mqtt_test.flux

```diff
@@ -0,0 +1,28 @@
+package mqtt_test
+
+
+import "array"
+import "experimental/mqtt"
+import "testing"
+
+testcase integration_mqtt_pub {
+    got =
+        array.from(
+            rows: [
+                {
+                    ok:
+                        mqtt.publish(
+                            broker: "tcp://127.0.0.1:1883",
+                            topic: "test/topic",
+                            message: "smoke test",
+                            qos: 0,
+                            retain: false,
+                            clientid: "fluxtest",
+                        ),
```

Comment:
This looks pretty straightforward, re our conversation in slack, did you end up playing any games with retain etc to ensure the message was published? Reading this test looks like we are relying on the client to report success/failure instead of validating through the server. Is that correct? 

This is likely good enough because we are using a standard client and we don't need to test if the server itself is working rather just that our client can successfully communicate with it.

Reply:
The issue I ran into is we have no API in flux to subscribe or otherwise read from MQTT, so there's no reasonable way to verify the outcome from within fluxtest alone. You'd need to be subscribes/watching in another process, or... something else.

Proving the dialer pooling behavior for that other issue might be a challenge, but as a manual exercise we can at least watch the logs on the server to look at client connects/disconnects.

For an automated test, the best I could come up with was to assert we got the `true` back from the client.

This test _does show an error if MQTT isn't running_, but I don't know what the rest of the failure space looks like :shrug:

Reply:
Addendum: I did use the spawner script to launch the broker and was able to watch (via a cli subscriber) messages hit the queue while using `mqtt.publish` from the REPL. (video in slack)